pymongo/collection.py

```diff
@@ -951,7 +951,22 @@ def update_one(
           {'x': 4, '_id': 0}
           {'x': 1, '_id': 1}
           {'x': 1, '_id': 2}
-
+          
+        If ``upsert=True`` and no documents match the filter, create a
+        new document based on the filter criteria and update modifications.
+        
+          >>> result = db.test.update_one({'x': -1*2**32}, {'$inc': {'x': 3}}, upsert=True)
```

Comment:
What do you think about using `update_one({'x': -10}, ...` instead (for clarity)?

Reply:
That makes sense to me. Fixed.